nixos/modules/services/hardware/display.nix

```diff
@@ -0,0 +1,141 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.hardware.display;
+in
+{
+  ###### interface
+
+  options = {
+    hardware.display.edid.packages = lib.mkOption {
+      type = with lib.types; listOf package;
+      default = [ ];
+      description = lib.mdDoc ''
+        List of packages containing EDID binary files at `$out/lib/firmware/edid`.
+        Such files will be available for use in `drm.edid_firmware` kernel
+        parameter as `edid/<filename>`.
+      '';
+      example = lib.literalExpression ''
+        [
+          (pkgs.runCommand "edid-custom" {} '''
+            mkdir -p "$out/lib/firmware/edid"
+            base64 -d > "$out/lib/firmware/edid/custom1.bin" <<'EOF'
+            <insert your base64 encoded EDID file here `base64 < /sys/class/drm/card0-.../edid`>
```

Comment:
We should provide an example how to download files like https://github.com/linuxhw/EDID/blob/master/Digital/ASUS/AUS3553/1F6B755EFEF5

Reply:
I've submitted a whole package doing that: https://github.com/NixOS/nixpkgs/pull/195815

Reply:
Should I pull it in maybe?

Reply:
I actually went ahead and pulled it in